src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -208,6 +208,7 @@ export class SearchResults extends Component {
         <h2
           className="va-introtext va-u-outline--none vads-u-font-size--lg vads-u-margin-top--1p5 vads-u-font-weight--normal"
           data-display-results-header
+          tabIndex="-1"
```

Comment:
Is the goal to have this focus last?

Reply:
Yeah, the focus is good, right after a search is fired. 
However, the tabindex is to make sure this is invisible in natural tab order of the page.